### DISCUSSION TITLE
Why do scores in the languages ratings do not match with profile honor?

### DISCUSSION THREAD

**Discussion by NiroTulip** [codewars/codewars.com#2984] [Q&A] [answered]
In the SQL rating I have 700 scores, while in the profile I have 634 honors, 468 of which are for completed SQL kata.
Why do scores in the SQL rating do not match with ny profile honor?
https://www.codewars.com/users/leaderboard/ranks?language=sql

![Scores](https://user-images.githubusercontent.com/89989049/231412303-23ad43b5-58a4-4cb5-9900-68922d52c6ea.jpg)

**Comment by Blind4Basics** [comment, marked as answer]
everything is there:
* https://docs.codewars.com/gamification/ranks
* https://docs.codewars.com/gamification/honor

cheers
